### PR TITLE
Added Upload Field

### PR DESCRIPTION
**Pull request by Tacchinardi20**
https://github.com/digi-serve/cars_app/issues/47

### CODE REVIEW

**Comment by Achoobert** [comment]
Lets add a test under `CARs test 1` `Test Adding New Child`
To open the `Admit Info` tab and look at the Asset Log detail.
It should contain today's date

'detail connected Initial Assets 6daf2109-4bbe-4003-b69d-bed120b83ec2 8db15d0b-fd78-4b2b-975a-304b68a43054'

**Comment by Achoobert** [comment]
LGTM

File: 0002_child_test.js [hunk 1]
```diff
@@ -559,12 +584,57 @@ describe("Test add-new forms:", () => {
...
+
+      cy.get(cyInterfaceCHILD.page.basicInfo.tab.admitInfo)
+         .should("exist")
+         .click();
+      cy.get('[data-cy="detail connected Initial Assets 6daf2109-4bbe-4003-b69d-bed120b83ec2 8db15d0b-fd78-4b2b-975a-304b68a43054"]')
+         .should("exist");
+      cy.get('[data-cy="menu-item Admission Info_3a46 cf368115-3a46-434b-b44e-6104b9bdb592 f076f340-9d27-45cf-b75e-c0930506235e"]')
+         .should("exist")
+         .click()
+         .wait(8000);  // Wait for the details showing
```
Comment (guyyoo) [inline 0002_child_test.js:610]: Should we use `cy.get(...).should("exist")` instead?

File: 0002_child_test.js [hunk 2]
```diff
@@ -575,19 +645,131 @@ describe("Test add-new forms:", () => {
...
+         '[data-cy="tab-SchoolRecords-ea8b6c3b-84f5-43ed-8f1c-65d0e73b0edc-d071e58a-baa8-4312-b8df-320b5f04c22a"]'
+      )
+         .should("exist")
+         .click();
+      cy.get(
+         '[data-cy="menu-item Add School Records 1eb85e88-a729-48fd-a7fc-cc1d3072f843 e520d8da-6e67-4b8f-a88f-2004c81ec813"]'
+      )
+         .should("exist")
+         .click()
+         .wait(2000); // Wait for finish loading
```
Comment (guyyoo) [inline 0002_child_test.js:681]: here too.

File: 0002_child_test.js [hunk 3]
```diff
@@ -575,19 +645,131 @@ describe("Test add-new forms:", () => {
...
+      cy.get('[data-cy="button save bf3dfa10-80a5-455a-869d-c981daa2cdb3"]')
+         .scrollIntoView()
+         .should("exist")
+         .contains("Save")
+         .click();
+      cy.get(".webix_warn")
+         .find(".webix_button")
+         .should("be.visible")
+         .click({ multiple: true, force: true })
+         .wait(4000); // Wait for finish loading
```
Comment (guyyoo) [inline 0002_child_test.js:708]: here too.

File: 0002_child_test.js [hunk 4]
```diff
@@ -575,19 +645,131 @@ describe("Test add-new forms:", () => {
...
+      });
+      cy.get(
+         '[data-cy="ABViewGrid_2fae0fda-4262-4ef0-b1c8-e91fe2fc1d82_datatable"]'
+      )
+         .should("be.visible")
+         .contains("2562");
+      cy.get(".wxi-pencil")
+         .should("be.visible")
+         .click({ force: true })
+         .wait(4000); // Wait for finish loading
```
Comment (guyyoo) [inline 0002_child_test.js:727]: here too.

File: 0002_child_test.js [hunk 5]
```diff
@@ -575,19 +645,131 @@ describe("Test add-new forms:", () => {
...
+         '[data-cy="list options 2563 79d81f68-2e41-47fd-a7d4-ec89ccc7dc58 3bf905ea-5a83-4412-a89e-5991325ae3a2"]'
+      )
+         .should("exist")
+         .click({ force: true });
+      cy.get('[data-cy="button save 3bf905ea-5a83-4412-a89e-5991325ae3a2"]')
+         .scrollIntoView()
+         .should("exist")
+         .contains("Save")
+         .click({ force: true })
+         .wait(2000); // Wait for finish loading
```
Comment (guyyoo) [inline 0002_child_test.js:751]: here too.

**Review by guyyoo — CHANGES_REQUESTED** [review]
A couple things.

File: 0002_child_test.js (hunk 1 above)
Comment (Tacchinardi20) [reply 0002_child_test.js]: Already Updated

**Comment by Tacchinardi20** [comment]
Already Updated

**Review by guyyoo — APPROVED** [review]
LGTM

**Pull request merged**